airflow.cfg

```diff
@@ -416,14 +416,14 @@ google_key_path =
 # headers can be used when making the actual request. This header is
 # the server side response to the browser's
 # Access-Control-Request-Headers header.
-access_control_allow_headers =
+access_control_allow_headers = origin, content-type, accept
 
 # Specifies the method or methods allowed when accessing the resource.
-access_control_allow_methods =
+access_control_allow_methods = POST, GET, OPTIONS, DELETE
 
 # Indicates whether the response can be shared with requesting code from the given origins.
 # Separate URLs with space.
-access_control_allow_origins =
+access_control_allow_origins = http://localhost:5432
```

Comment:
this seems hardcoded and will not work for containers, maybe just`*` for development?